pass-manager-svc/pass-owner/src/main/kotlin/com/example/passmanagersvc/passowner/application/port/service/PassOwnerService.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.passmanagersvc.passowner.application.port.service
```

Comment:
```suggestion
package com.example.passmanagersvc.passowner.application.service
```

please fix all places